app/dashboard/views.py

```diff
@@ -3609,17 +3609,25 @@ def get_kudos(request):
     mimetype = 'application/json'
     return HttpResponse(data, mimetype)
 
-
+@login_required
 def hackathon_prizes(request, hackathon=''):
-    hackathon_event = get_object_or_404(HackathonEvent,
-                                        slug__iexact=hackathon)
-
-    is_founder = Bounty.objects.filter(event=hackathon_event, bounty_owner_profile=request.user.profile)
-
-    if not is_founder:
+    profile = request.user.profile
+    hackathon_event = get_object_or_404(HackathonEvent, slug__iexact=hackathon)
+    sponsors = hackathon_event.sponsor_profiles.all().values_list('id', flat=True)
+    is_sponsor_member = profile.organizations_fk.filter(pk__in=sponsors)
+    funded_bounties = profile.bounties_funded.filter(event=hackathon_event)
+
+    if profile.is_staff:
+        query_prizes = Bounty.objects.filter(event=hackathon_event)
+    elif is_sponsor_member.exists():
+        sponsor = is_sponsor_member.first()
+        query_prizes = Bounty.objects.filter(event=hackathon_event, bounty_owner_profile=sponsor)
```

Comment:
I wonder if bounty_owner_profile will the org name? or is always a user profile (not org) ?

Reply:
![image](https://user-images.githubusercontent.com/1358093/88825046-79196700-d19d-11ea-87c2-ebe2c2a66309.png)
example this is from gitcoin repo but the owner is my personal profile, this query will return prizes?

Reply:
I added one more commit to manage this scenario:
- Will check if you are a member of one sponsor, if so it will display all prizes related to the sponsor.

Reply:
Good catch! :+1:

Reply:
we really need to add bounty org profile and populate it, asap.